docs/how_jstz_works.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 👨‍⚖️ How jstz works?
+---
+
+<script setup>
+import VPButton from "vitepress/dist/client/theme-default/components/VPButton.vue";
+</script>
+
+# 👨‍⚖️ `How jstz works?`
+
+## Overview
+
+`jstz` is a JavaScript/Typescript server runtime for Tezos 2.0 designed to provide a great developer experience by aiming to be compatible with web conventions.
+
+Through `jstz` developers can set up, deploy and test so called _smart functions_ written in Javascript/Typescript that can get directly executed on Tezos L2.
+
+Jstz provides a local sandboxed environment for developers to test the functions without deploying them to L2. It provides a simple interface through which one can deploy smart functions and then call them by sending HTTP requests to a particular _smart function address_.
```

Comment:
Jstz->`jstz`. 
Some grammar changes.
Moved running of smart functions to above paragraph

```suggestion
`jstz` also provides a local sandboxed environment for developers to test their functions without deploying them to production.
```